### ISSUE TITLE
create-sibling-ria: fatal: repository not found

### ISSUE THREAD

**Issue by mih** [datalad/datalad#5530]
I create a new sibling in an existing store, and it fails:

```
% datalad create-sibling-ria -s dataladstore --post-update-hook --push-url ria+ssh://store.datalad.org/srv/store.datalad.org/www ria+http://store.datalad.org
[INFO   ] create siblings 'dataladstore' and 'dataladstore-storage' ... 
[INFO   ] Fetching updates for Dataset(/home/mih/hacking/datalad/paper-joss) 
CommandError: 'git -c diff.ignoreSubmodules=none fetch --verbose --progress --prune --recurse-submodules=no dataladstore' failed with exitcode 128 under /home/mih/hacking/datalad/paper-joss
fatal: repository 'http://store.datalad.org/a49/6f0ce-1f4a-4720-b599-3c1ce3aacaf7/' not found
```

Neither `git clone`, not `git fetch` work.

```
% git fetch --all --prune
Fetching origin
remote: Enumerating objects: 9, done.
remote: Counting objects: 100% (9/9), done.
remote: Compressing objects: 100% (5/5), done.
remote: Total 7 (delta 4), reused 4 (delta 2), pack-reused 0
Unpacking objects: 100% (7/7), 2.96 KiB | 202.00 KiB/s, done.
From github.com:datalad/datalad-paper-joss
   004c2f0..c923d19  master     -> origin/master
Fetching dataladstore
fatal: repository 'http://store.datalad.org/a49/6f0ce-1f4a-4720-b599-3c1ce3aacaf7/' not found
error: Could not fetch dataladstore
```

But I can `datalad push` to the storage sibling:

```
% datalad push --to dataladstore-storage 
...
copy(ok): .datalad/environments/openjournals-paperdraft/image/2ef067356d99edfdb5d78bf995cd03dde5fd9a4c270ff8238d1d941ca7f9b3f1/json (file) [to dataladstore-storage...]
  [28 similar messages have been suppressed]                                                                               
datalad push --to dataladstore-storage  70.54s user 14.61s system 1% cpu 1:26:22.17 total                                  
```

I can also `datalad push` to the git remote:

```
% datalad push --to dataladstore
publish(ok): . (dataset) [refs/heads/localdraft->dataladstore:refs/heads/localdraft [new branch]]                          
publish(ok): . (dataset) [refs/heads/git-annex->dataladstore:refs/heads/git-annex [new branch]]                            
```

Afterwards, I can fetch just fine.

```
% git fetch --all
Fetching origin
Fetching dataladstore
```

I think this puts the blame on `create-sibling-ria`.

**Comment by bpoldrack** [comment]
`create-sibling-ria` misses a call to  `git update-server-info` after setting up the repo!

**Comment by mih** [comment]
Fixed by #5531
